final-project/manJoinProcess.tex

```diff
@@ -63,4 +63,16 @@ \subsection{Messaging Protocol}
 
 \section{Creating or Loading Rooms}
 
+The client will have two room creation options: generating a temporary room with a random token and establishing a permanent room that can be rejoined in the future. 
+
+Both creation options follow similar paths, as shown in figure \ref{Figure::create-room-diag}. The client starts a new Balancer connection through the websocket, then the Balancer is responsible for deciding which node is best fit to handle the additional room. Once the right one is picked, the room is instantiated on that Monolith and the client connects.
+
+The main difference lies in what happens after the initial usage of the room. With a temporary room, after the clients disconnect the room is deleted from the Balancer hashmap of Monoliths and rooms. With a permanent room, the unique token is maintained and can be hosted again at any point in the future.
```

Comment:
The "token" terminology can be confused with an auth token, which is not related here.

In a permanent room, the room settings are persisted in postgres so that when the room is loaded again, the settings remain intact. With a temporary room, when the room is unloaded, the room is simply thrown away.